From d968ff96e1e4354c591e49fe79b4f4ce0e553f61 Mon Sep 17 00:00:00 2001
From: Robby Workman <rworkman@slackware.com>
Date: Sun, 11 Aug 2013 23:12:57 -0500
Subject: [PATCH 2/2] dhcpcd.conf: Don't invoke wpa_supplicant by default

Slackware's network init script (rc.inet1) and/or
NetworkManager itself handles this just fine.
---
 dhcpcd.conf | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/dhcpcd.conf b/dhcpcd.conf
index 349c6f2..30aacca 100644
--- a/dhcpcd.conf
+++ b/dhcpcd.conf
@@ -30,3 +30,7 @@ require dhcp_server_identifier
 # A hook script is provided to lookup the hostname if not set by the DHCP
 # server, but it should not be run by default.
 nohook lookup-hostname
+
+# We run wpa_supplicant from rc.inet1 and/or NM handles it on its own
+nohook wpa_supplicant
+
-- 
1.8.3.2

